spec/helpers/application_helper_spec.rb

```diff
@@ -18,6 +19,15 @@
       expect(current_controller?(:baz, :bar)).to be_falsy
       expect(current_controller?(:baz, :bar, :foo)).to be_truthy
     end
+
+    it "should take namespace into account" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.